package.json

```diff
@@ -70,48 +70,47 @@
     "deep-freeze": "0.0.1",
     "js-yaml": "3.12.0",
     "lodash": "4.17.11",
-    "patternfly": "3.48.3",
-    "patternfly-react": "2.20.3",
-    "react": "16.5.2",
-    "react-ace": "6.1.1",
-    "react-bootstrap": "0.32.1",
-    "react-dom": "16.5.2",
+    "patternfly": "3.58.0",
+    "patternfly-react": "2.25.3",
+    "react": "16.6.3",
+    "react-ace": "6.2.0",
+    "react-bootstrap": "0.32.4",
+    "react-dom": "16.6.3",
     "react-draggable": "3.0.5",
-    "react-flexview": "4.0.0",
-    "react-floater": "^0.6.0",
-    "react-iframe": "1.1.0",
-    "react-redux": "5.0.7",
-    "react-resize-detector": "3.0.1",
+    "react-flexview": "4.0.1",
+    "react-floater": "0.6.1",
+    "react-iframe": "1.3.3",
+    "react-redux": "5.1.1",
+    "react-resize-detector": "3.2.1",
     "react-router-dom": "4.3.1",
-    "react-scripts-ts": "2.15.1",
-    "redux": "4.0.0",
+    "react-scripts-ts": "4.0.8",
+    "redux": "4.0.1",
     "redux-mock-store": "1.5.3",
     "redux-persist": "5.10.0",
     "redux-thunk": "2.3.0",
-    "reselect": "3.0.1",
-    "ssri": "6.0.0",
+    "reselect": "4.0.0",
+    "ssri": "6.0.1",
     "typesafe-actions": "2.0.4",
     "typestyle": "2.0.1",
-    "url-search-params": "1.1.0",
     "visibilityjs": "2.0.2"
   },
   "devDependencies": {
-    "@types/enzyme": "3.1.14",
-    "@types/jest": "23.3.5",
-    "@types/lodash": "4.14.116",
-    "@types/react": "16.4.17",
-    "@types/react-dom": "16.0.9",
-    "@types/react-redux": "6.0.9",
+    "@types/enzyme": "3.1.15",
+    "@types/jest": "23.3.10",
+    "@types/lodash": "4.14.119",
+    "@types/node": "10.12.11",
+    "@types/react": "16.7.13",
+    "@types/react-dom": "16.0.11",
+    "@types/react-redux": "6.0.11",
     "@types/react-router-dom": "4.3.1",
-    "@types/url-search-params": "0.10.1",
     "axios-mock-adapter": "1.15.0",
     "babel-core": "6.26.3",
-    "enzyme": "3.3.0",
-    "enzyme-adapter-react-16": "1.6.0",
-    "husky": "1.2.0",
-    "jest-canvas-mock": "1.0.2",
-    "jest-localstorage-mock": "2.2.0",
-    "node-sass-chokidar": "1.3.0",
+    "enzyme": "3.8.0",
+    "enzyme-adapter-react-16": "1.7.1",
+    "husky": "0.14.3",
```

Comment:
bumping back husky from 1.2.0 to 0.14.3 ? Looks weird, is it a mistake or is there a good reason?

Reply:
Not a mistake, Husky version 1.2.0 didn't work, as it was a major version change and needed more work that wasn't that relevant to the React and PF upgrades. So backed it out and created a jira for it: https://issues.jboss.org/browse/KIALI-2158 which was merged https://github.com/kiali/kiali-ui/pull/890